src/main/java/com/internetitem/logback/elasticsearch/ClassicElasticsearchPublisher.java

```diff
@@ -1,18 +1,14 @@
 package com.internetitem.logback.elasticsearch;
 
-import java.io.IOException;
-import java.util.Map;
-
 import ch.qos.logback.classic.spi.ILoggingEvent;
 import ch.qos.logback.core.Context;
 import com.fasterxml.jackson.core.JsonGenerator;
-import com.internetitem.logback.elasticsearch.config.ElasticsearchProperties;
-import com.internetitem.logback.elasticsearch.config.HttpRequestHeaders;
-import com.internetitem.logback.elasticsearch.config.Property;
-import com.internetitem.logback.elasticsearch.config.Settings;
-import com.internetitem.logback.elasticsearch.util.AbstractPropertyAndEncoder;
-import com.internetitem.logback.elasticsearch.util.ClassicPropertyAndEncoder;
-import com.internetitem.logback.elasticsearch.util.ErrorReporter;
+import com.internetitem.logback.elasticsearch.config.*;
+import com.internetitem.logback.elasticsearch.util.*;
+
+import java.io.IOException;
+import java.util.*;
```

Comment:
New to the project, but I don't think star-imports are used anywhere else (and haven't been used before in this file).